packages/navstar/build

```diff
@@ -5,7 +5,7 @@ source /opt/mesosphere/environment
 
 export CFLAGS="-I/opt/mesosphere/include -I/opt/mesosphere/active/libsodium/include"
 export LDFLAGS="-L/opt/mesosphere/lib -L/opt/mesosphere/active/libsodium/lib -Wl,-rpath=/opt/mesosphere/active/libsodium/lib"
-export CXXFLAGS=CFLAGS
+export CXXFLAGS=${CFLAGS}
```

Comment:
Ah, I got this from 74760244. Should I change it across the board in another PR?